SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -750,7 +780,11 @@ void USpatialReceiver::DestroyActor(AActor* Actor, Worker_EntityId EntityId)
 	}
 	else
 	{
-		if (Actor == nullptr)
+		if (NetDriver->IsDormantEntity(EntityId))
+		{
+			NetDriver->PackageMap->RemoveEntityActor(EntityId);
```

Comment:
I think the actor associated with this entity doesn't get destroyed. Could you please check?